modules/docs/mdx/12.0-UPGRADE-GUIDE.mdx

```diff
@@ -15,7 +15,7 @@ any questions.
 
 ## Why You Should Upgrade
 
-Canvas Kit v12 is transitioning into a
+Canvas Kit is transitioning into a
 [new way of styling](https://github.com/Workday/canvas-kit/discussions/2265). Theming and building
```

Comment:
Not sure we still need this text? This was copied over from v11, do you think we should keep it around?

Reply:
Yup we'll leave that in as since we're still moving components over to our new styling utils and tokens.